kernel/reduction.ml

```diff
@@ -153,13 +153,13 @@ let rec test (sg:Signature.t) (convertible:convertibility_test)
     then test sg convertible ctx tl
     else false
   | (Bracket (i,t2))::tl ->
-    let t1 = mk_DB dloc dmark i in
-    if convertible sg (term_of_state { ctx; term=t1; stack=[] })
-                      (term_of_state { ctx; term=t2; stack=[] })
+    let t1 = Lazy.force (LList.nth ctx i) in
+    let t2 = term_of_state { ctx; term=t2; stack=[] } in
+    if convertible sg t1 t2
```

Comment:
It is a bad idea to have a variable which depend of another variable, having the same name.